lnd_test.go

```diff
@@ -44,6 +44,7 @@ var (
 
 const (
 	testFeeBase         = 1e+6
+	defaultCSV          = 4
```

Comment:
similar comment here wrt placement in `lntest`. @wpaulino also pointed out that we pass in a defaultcsv value to the node via cli args when starting, this value can be `Sprintf`'d into the args as well